packages/session-reducer/src/index.ts

```diff
@@ -165,6 +210,25 @@ export function getAccessToken(state: Partial<Store>): string | null {
   return null;
 }
 
+/** check if token is updated as expired
+ * @param {Partial<Store>} state - the redux store
+ */
+export function isTokenExpired(state: Partial<Store>): boolean {
+  return !getAccessToken(state) ? true : getTokenExiryStatus(state) === TokenStatus.expired;
+}
+
+/**
+ * get token or redirect to session expired page
+ * @param {Partial<Store>} state - the redux store
+ */
+export const getTokenOrRedirect = (state: Partial<Store>, redirectTo: string | null = null) => {
```

Comment:
Do you think that the session reducer package needs to be the entity that determines what happens when a session is not valid?  Why?

Please try to change this opinion that I currently hold: the session reducer package does not care what you do, it just gives you the token, or tells you that the current token is invalid.